packages/core/src/parsers/linkProcessor.js

```diff
@@ -45,6 +45,19 @@ function convertRelativeLinks(node, cwf, rootPath, baseUrl) {
   node.attribs[linkAttribName] = path.posix.join(baseUrl || '/', resourcePathFromRoot);
 }
 
+function convertMdExtToHtmlExt(node) {
+  if (node.attribs && node.attribs.href) {
+    const { href } = node.attribs;
+    const hasMdExtension = href && href.slice(-3) === '.md';
```

Comment:
first `href &&` can be removed since you checked it above already

Reply:
ah yeah, thanks for highlighting this, I overlooked it!